packages/conf-pkg-config/conf-pkg-config.1.1/opam

```diff
@@ -5,13 +5,8 @@ homepage: "http://www.freedesktop.org/wiki/Software/pkg-config/"
 bug-reports: "https://github.com/ocaml/opam-repository/issues"
 license: "GPL"
 build: [
-  ["pkg-config" "--help"]
-]
-install: [
   ["ln" "-s" "/usr/local/bin/pkgconf" "%{bin}%/pkg-config"] {os = "openbsd"}
```

Comment:
this definitely shouldn't work when ran during the build phase (with sandboxing on)